scipy/stats/_continuous_distns.py

```diff
@@ -1555,7 +1555,13 @@ def _logpdf(self, x, c):
         return log(c) + special.xlogy(c - 1, x) - pow(x, c)
 
     def _cdf(self, x, c):
-        return -special.expm1(-pow(x, c))
+        return -np.expm1(-pow(x, c))
```

Comment:
Argh, I forgot about that, and I even commented on the numpy issue!  I'll go back to using the `special` versions.